raster/r.info/main.c

```diff
@@ -110,6 +120,17 @@ int main(int argc, char **argv)
     if ((mapset = G_find_raster2(name, "")) == NULL)
         G_fatal_error(_("Raster map <%s> not found"), name);
 
+    if (strcmp(fopt->answer, "json") == 0)
+        format = JSON;
+    else
+        format = PLAIN;
+
+    if (format == JSON) {
+        root_value = json_value_init_object();
+        root_object = json_value_get_object(root_value);
+        json_set_float_serialization_format("%lf");
```

Comment:
@kritibirda26 why did you set ` json_set_float_serialization_format("%lf")` instead of using parson's default value `%1.17g`?

Reply:
I saw some other `fpritnt`s for plain output were using `%lf` format specifier for floats, hence I changed it for JSON too.

Reply:
Okay, lets just use the parson default to be consistent across all of the modules. This will still remove trailing zeros, but that is one of the issues with JSON.

How about we use add the json schema to tests so if we can preserve types there.

Reply:
I will update the format specifier. Can you please elaborate what you mean by adding the json schema to tests?